gammapy/irf/effective_area_table.py

```diff
@@ -444,6 +446,67 @@ def read(filename):
         hdu_list = fits.open(filename)
         return EffectiveAreaTable2D.from_fits(hdu_list)
 
+    def to_effectiveareatable(self, offset, energ_lo = None, energ_hi = None):
+        """Evaluate at a given offset and return effective area table object.
+
+        The energy thresholds in the effective area table object are not set.
+        If the effective area table is intended to be used for spectral analysis,
+        the final energy binning should be given at this point, since the
+        effective area table class is not able to perform interpolation at
+        this point.
+
+        Parameters
+        ----------
+        offset : `~astropy.coordinates.Angle`
+            offset
+        energy : `~astropy.units.Quantity`
+            energy
+
+        Returns
+        -------
+        eff_area_table : `EffectiveAreaTable`
+             Effective area table class. 
+        """
+
+        if energ_lo is None and energ_lo is None:
+            energ_lo = self.energ_lo  
+            energ_hi = self.energ_hi
+        elif energ_lo is None or energ_hi is None:
+            raise ValueError("Only 1 energy vector given, need 2")
+        if not isinstance(offset, Angle):
+            raise ValueError("Offset must be an angle object.")
+        if not isinstance(energ_lo, Quantity) or not isinstance(energ_hi, Quantity):
+            raise ValueError("Energy must be a Quantity object.")
+        if len(energ_lo) != len(energ_hi):
+            raise ValueError("Energy Vectors must have same dimension")
+        
+        energy = np.sqrt(energ_lo * energ_hi)
+        area = self.evaluate(offset,energy) 
+
+        return EffectiveAreaTable(energ_lo, energ_hi, area)
+
+    def write_ARF(self, filename, offset, energ_lo = None, energ_hi = None):
+        """Write ARF Fits file 
+
+        Uses the write method of the effective area table class
+        
+        Parameters
+        ----------
+        filename: str
+            output file name        
+        offset : `~astropy.coordinates.Angle`
+            offset
+        energy : `~astropy.units.Quantity`
+            energy
+
+        Returns
+       -------
+        arf : `EnergyDependentARF`
+            ARF object.
+        """
```

Comment:
This docstring gives a Sphinx warning, which makes the travis-ci build fail (i.e. this PR can't be merged):
https://travis-ci.org/gammapy/gammapy/jobs/65093013#L1843

I'm not sure what the issue is. Maybe the missing space between `filename` and colon in `filename:`?

Debugging / fixing Sphinx warnings and errors is tedious (or I'm simply not aware of a good method).
What I do is usually remove things one by one until the error goes away and then when I know where the issue is I can usually guess or Google how to fix it.